storage/src/main/java/zipkin2/storage/kafka/internal/Traces.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 jeqo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin2.storage.kafka.internal;
+
+import java.util.List;
+import zipkin2.Span;
+
+public class Traces {
```

Comment:
naive wrapper to parse traces with gson

Reply:
remind me why traces are stored as nested list? pointing me to code is also fine

Reply:
Not to store but to serialize response here: https://github.com/jeqo/zipkin-storage-kafka/blob/3e831a269980d106ae474a92d1a0343792130bf3/storage/src/main/java/zipkin2/storage/kafka/KafkaStoreServerSupplier.java#L286-L289

Reply:
probably less janky than this I guess :P https://github.com/openzipkin/zipkin/blob/master/zipkin-server/src/main/java/zipkin2/server/internal/ZipkinQueryApiV2.java#L191

Reply:
I think there's probably a nice pooled buffer variant for this if/when we port to jackson cc @anuraaga

Reply:
have moved it to jackson and no need for this wrapper now :)